src/repositories/maptoWirelessProfile.ts

```diff
@@ -0,0 +1,20 @@
+
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2019
+ * SPDX-License-Identifier: Apache-2.0
+ * Author : Madhavi Losetty
+ **********************************************************************/
+
+import { WirelessConfig } from '../RCS.Config'
+
+export function mapToWirelessProfile (results): WirelessConfig {
+  return {
```

Comment:
just a suggestion, might be better to def a variable typed as WirelessConfig instead of casting.

Reply:
done